tests/src/test/scala/ironrust/FullIntegrationTest.scala

```diff
@@ -8,55 +8,36 @@ import scodec.bits.ByteVector
 
 class FullIntegrationTest extends DudeSuite with CancelAfterFailure {
   //Generates a random user ID and password to use for this full integration test
-  val primaryTestUserId = Try(UserId.validate(java.util.UUID.randomUUID().toString())).toEither.value
-  val primaryTestUserPassword = java.util.UUID.randomUUID().toString()
+  val primaryTestUserId = Try(UserId.validate(java.util.UUID.randomUUID.toString)).toEither.value
+  val primaryTestUserPassword = java.util.UUID.randomUUID.toString
 
-  //Stores record of integration test users device context parts that are then used to initialize the
-  //SDK for each test
+  //Stores record of integration test users device context parts that are then used to initialize the SDK
   var primaryTestUserSegmentId = 0L
   var primaryTestUserPrivateDeviceKeyBytes: Array[Byte] = null
   var primaryTestUserSigningKeysBytes: Array[Byte] = null
 
-  val secondaryTestUserId = Try(UserId.validate(java.util.UUID.randomUUID().toString())).toEither.value
-  val secondaryTestUserPassword = java.util.UUID.randomUUID().toString()
+  val secondaryTestUserId = Try(UserId.validate(java.util.UUID.randomUUID.toString)).toEither.value
+  val secondaryTestUserPassword = java.util.UUID.randomUUID.toString
   var secondaryTestUserSegmentId = 0L
   var secondaryTestUserPrivateDeviceKeyBytes: Array[Byte] = null
   var secondaryTestUserSigningKeysBytes: Array[Byte] = null
 
   var validGroupId: GroupId = null
   var validDocumentId: DocumentId = null
 
-  val defaultPolicyCaching = new PolicyCachingConfig()
+  val defaultPolicyCaching = new PolicyCachingConfig
 
   val shortTimeout = Duration.from_millis(5)
   val defaultTimeout = Duration.from_secs(30)
   val longTimeout = Duration.from_secs(30)
 
   val shortConfig = new IronOxideConfig(defaultPolicyCaching, shortTimeout)
-  val defaultConfig = new IronOxideConfig()
-
-  /**
-    * Convenience function to create a new DeviceContext instance from the stored off components we need. Takes the
-    * users account ID, segment ID, private device key bytes, and signing key bytes and returns a new DeviceContext
-    * instance. This helps us prove that we can create this class instance from scratch.
-    */
-  def createDeviceContext = {
-    new DeviceContext(
-      primaryTestUserId,
-      primaryTestUserSegmentId,
-      PrivateKey.validate(primaryTestUserPrivateDeviceKeyBytes),
-      DeviceSigningKeyPair.validate(primaryTestUserSigningKeysBytes)
-    )
-  }
+  val defaultConfig = new IronOxideConfig
 
-  def createSecondaryDeviceContext = {
-    new DeviceContext(
-      secondaryTestUserId,
-      secondaryTestUserSegmentId,
-      PrivateKey.validate(secondaryTestUserPrivateDeviceKeyBytes),
-      DeviceSigningKeyPair.validate(secondaryTestUserSigningKeysBytes)
-    )
-  }
+  var deviceContext: DeviceContext = null
+  var secondaryDeviceContext: DeviceContext = null
+
+  var sdk: IronOxide = null
```

Comment:
Correct. 
And I agree that tests being able to be run in parallel is the goal, but it's not where these tests are right now. So I think that since the tests are currently run in series, they might as well run faster in series.